src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -121,7 +121,8 @@ void fillInnerParts() {
         resultDisplay = new ResultDisplay();
         resultDisplayPlaceholder.getChildren().add(resultDisplay.getRoot());
 
-        StatusBarFooter statusBarFooter = new StatusBarFooter(logic.getAddressBookFilePath(), logic.getAddressBook());
+        StatusBarFooter statusBarFooter = new StatusBarFooter(logic.getAddressBookFilePath(), logic.getAddressBook(),
+                logic.getFilteredPersonList());
```

Comment:
Huh, I'm surprised this passes checkstyle. Oh well :P

Reply:
This line breaking is to stay consistent with a few lines before. Although a readability issue, maybe we fix in another PR?